tests/test-bobfiles/regenerate_test_bobfiles.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+from fixtures.mocked_panda import (
+    BOBFILE_DIR,
+    TEST_PREFIX,
+    ResponseHandler,
+    create_subprocess_ioc_and_responses,
+)
+
+
+@pytest.fixture
+def mocked_panda_generate_new_bobfiles(
+    standard_responses,
+    caplog,
+    caplog_workaround,
+    table_field_info,
+    table_fields,
+):
+    response_handler = ResponseHandler(standard_responses)
+
+    yield from create_subprocess_ioc_and_responses(
+        response_handler,
+        BOBFILE_DIR,
+        TEST_PREFIX,
+        caplog,
+        caplog_workaround,
+        table_field_info,
+        table_fields,
+        clear_bobfiles=True,
+    )
+
+
+def test_regenerate_test_bobfiles(mocked_panda_generate_new_bobfiles):
```

Comment:
Probably worth adding a comment here to say this test isn't really a test and is just for easily regenerating all bobfiles. Or that might be overkill, given the name of the file and the test. I'm just generally in the overdocument, rather than underdocument, camp.

Reply:
Agreed